hack/update-codegen.sh

```diff
@@ -16,7 +16,13 @@
 set -o errexit
 set -o nounset
 set -o pipefail
+set -o xtrace
 
+echo "Pre-existing \$GOPATH: ${GOPATH}"
```

Comment:
You need ${GOPATH;-} else this will blow up because of `nounset`, right?

Reply:
This is debug logging to figure out why my script is getting `$GOPATH=/go`